### PR TITLE
add description to manager components

### PR DESCRIPTION
**Pull request by bsoutham**
add description to blade manager ("HPE Integrated Lights Out") and to enclosure manager ( "HPE Onboard Administrator")
modify tests to pass

### CODE REVIEW

File: oneview_redfish_toolkit/api/blade_manager.py [hunk 1]
```diff
@@ -39,9 +39,9 @@ def __init__(self, server_hardware):
 
         super().__init__(server_hardware, server_hardware['mpFirmwareVersion'])
 
-        self.redfish["FirmwareVersion"] = server_hardware['mpFirmwareVersion']
```
Comment (danielreed) [inline oneview_redfish_toolkit/api/blade_manager.py]: Why was this removed?

File: oneview_redfish_toolkit/api/blade_manager.py [hunk 2]
```diff
@@ -39,9 +39,9 @@ def __init__(self, server_hardware):
...
         self.redfish["ManagerType"] = "BMC"
         self.redfish["Name"] = "Blade Manager"
+        self.redfish["Description"] = "HPE Integrated Lights Out"
```
Comment (danielreed) [inline oneview_redfish_toolkit/api/blade_manager.py]: This isn't really the iLO that we are exposing in this service.  Shouldn't the description be something more like 'Synergy Blade Manager' or 'HPE OneView Blade Manager'?

File: oneview_redfish_toolkit/api/enclosure_manager.py [hunk 3]
```diff
@@ -41,6 +41,7 @@ def __init__(self, enclosure, oneview_version):
 
         self.redfish["ManagerType"] = "EnclosureManager"
         self.redfish["Name"] = "Enclosure Manager"
+        self.redfish["Description"] = "HPE Onboard Administrator"
```
Comment (danielreed) [inline oneview_redfish_toolkit/api/enclosure_manager.py]: This isn't really the OA that we are exposing in this service. Shouldn't the description be something more like 'Synergy Manager' or 'HPE OneView Manager'?

File: oneview_redfish_toolkit/api/blade_manager.py (hunk 1 above)
Comment (bsoutham) [reply oneview_redfish_toolkit/api/blade_manager.py]: it is passed into the super class and stored.  This is a duplicate and should have been removed.

File: oneview_redfish_toolkit/api/blade_manager.py (hunk 2 above)
Comment (bsoutham) [reply oneview_redfish_toolkit/api/blade_manager.py]: If the manager type is a BMC (baseboard managmenet controller) -- then for HPE gear, that iLO.  Other vendors have other names (BMC, DRAC, etc.)

https://hewlettpackard.github.io/ilo-rest-api-docs/ilo5/?python#find-the-ilo-5-management-processor
seems to indicate that iLO uses them interchangeably.


Are you saying that the description is for the logical redfish manager, and not the underlying hardware component?

File: oneview_redfish_toolkit/api/enclosure_manager.py (hunk 3 above)
Comment (bsoutham) [reply oneview_redfish_toolkit/api/enclosure_manager.py]: same comment as above
